README.md

```diff
@@ -1,56 +1,38 @@
-development environment.
 
-*  Provide users a tool to control and configure the ADB from the
-developer's workstation without having to `ssh` into it.
+## Objective
 
-The [Atomic Developer
-Bundle](https://github.com/projectatomic/adb-atomic-developer-bundle)
-is  Vagrant box that provides a ready-to-use development environment
-for container applications. With ADB, developers can dive right into
+The [ADB](https://github.com/projectatomic/adb-atomic-developer-bundle) provides a ready-to-use development environment for container applications. With ADB, developers can dive right into
 producing complex, multi-container applications.
 
-## Quick Start
+The vagrant-service-manager provides the user with:
 
-1. Install `vagrant-service-manager` plugin:
+* A CLI to configure the ADB for different use cases and to provide an interface between ADB and the user's development environment.
 
-        vagrant plugin install vagrant-service-manager
-
-2. Get a Vagrantfile for your box. Users of the
-[Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle) should download a [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components).
-
-3. Enable your desired service(s) in [Vagrantfile](Vagrantfile) as:
-
-        config.servicemanager.services = 'openshift'
-
-   *Note*
+* A tool to control and configure the ADB from the
+developer's workstation without having to `ssh` directly into the ADB virtual machine.
 
-   * `docker` is default service and does not require above configuration.
-   * Enable multiple services as comma separated list. Eg: 'docker, openshift'
 
-4. Enable any specific options for the services you have selected:
+## Example execution of the plugin
 
-    * OpenShift
+1. Install vagrant-service-manager plugin:
 
-        Specific versions can be specified using the following variables:
-
-        `config.servicemanager.openshift_docker_registry = "docker.io"` - What registry should be pulled from
-        `config.servicemanager.openshift_image_name = "openshift/origin"` - What image should be used
-        `config.servicemanager.openshift_image_tag = "v1.1.1"` - What image version should be used
+        vagrant plugin install vagrant-service-manager
 
+2. Get a Vagrantfile for your box of choice. Users of the
+[ADB](https://github.com/projectatomic/adb-atomic-developer-bundle) should download the relevant [Vagrantfile from the repository](https://github.com/projectatomic/adb-atomic-developer-bundle/tree/master/components/centos). For further details and to use custom vagrant files designed for specific use cases, refer to the [usage documentation](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.rst).
 
-5. Start the ADB using `vagrant up`. Users of the ADB may wish to consult the
+3. Start the vagrant box or ADB using `vagrant up`. For detailed instructions consult the
```

Comment:
yes I have seen that before. But does VSM have an use case  outside ADB and CDK ? Then why confuse users saying that it can be used outside of ADB/CDK @bexelbie @navidshaikh